### PR TITLE
Add a note about RouteDefaultsValidator

### PR DESCRIPTION
**Pull request by EmmanuelVella**
Refs #667 

@dbu I'm don't know what you exactly want to document here, so I just added a small note about it.

### CODE REVIEW

**Comment by dbu** [comment]
thanks!
i thought we could explain the validator for people that write their own route model. the validator itself is not specific to phpcr-odm. and then this note would just mention that its using the validator. the validator doc could go into dynamic_customize.rst because its only relevant when doing your own stuff.

File: bundles/routing/dynamic.rst [hunk 1]
```diff
@@ -278,6 +278,11 @@ Of course you can also have several parameters, as with normal Symfony
 routes. The semantics and rules for patterns, defaults and requirements are
 exactly the same as in core routes.
 
+.. note::
+
+    The ``RouteDefaultsValidator`` validates the route defaults parameters.
+    For more information, see :ref:`<_bundle-routing-route-defaults-validator>`.
```
Comment (dbu) [inline bundles/routing/dynamic.rst]: you need to write the ref without the leading `_` for sphinx to find the target

**Comment by EmmanuelVella** [comment]
@dbu Thanks ! Is it ok or do you want me to add something else ?

**Comment by dbu** [comment]
thanks!

**Pull request merged**